hm_pyhelper/miner_param.py

```diff
@@ -191,6 +191,40 @@ def get_ethernet_addresses(diagnostics):
             raise MinerFailedToFetchEthernetAddress(str(e))
 
 
+def get_temperatures(diagnostics):
+    path_to_temp_file = "/sys/class/thermal/thermal_zone0/temp"
+
+    key = "CPU0"
+
+    try:
+        file = open(path_to_temp_file)
+    except FileNotFoundError as e:
+        LOGGER.exception("Failed to find Miner"
+                         "Temperature file at path %s" % path_to_temp_file)
+        diagnostics[key] = False
```

Comment:
When you refactor [into a Diagnostic](https://github.com/NebraLtd/hm-diag/pull/339#discussion_r857671613), I suggest you make this method return the temperature, instead of mutating a diagnostic dict that is passed in. In that case, you will only throw an exception on error, and will not set anything to False in case of error.

Reply:
Sorry but I do not have the Python skills. I just followed how the others were.